api/test/generate-data/factories/organization_factory.js

```diff
@@ -48,9 +48,9 @@ factory.define(factoryName, Organization, buildOptions => {
     , companyLegal: ""
     , company: companyName
 
-    , read             : faker.random.arrayElement(["public", "sysadmin", ["public", "sysadmin"]])
-    , write            : faker.random.arrayElement(["public", "sysadmin", ["public", "sysadmin"]])
-    , delete           : faker.random.arrayElement(["public", "sysadmin", ["public", "sysadmin"]])
+    , read             : faker.random.arrayElement(["staff", "sysadmin", ["staff", "sysadmin"]])
```

Comment:
@KitArmstrong why did this change?

Reply:
This was updated to match the roles that are used in the controller when an Organization is created.